x-pack/legacy/plugins/task_manager/mappings.json

```diff
@@ -16,8 +16,12 @@
       "retryAt": {
         "type": "date"
       },
-      "interval": {
-        "type": "text"
+      "schedule": {
+        "properties": {
+          "interval": {
+            "type": "keyword"
+          }
+        }
```

Comment:
Issue I saw when starting up to test this was:

```sh
log   [07:35:05.285] [fatal][root] { Error: mapping set to strict, dynamic introduction of [interval] within [alert] is not allowed
    at Object.write (/Users/frankhassanabad/projects/kibana/src/core/server/saved_objects/migrations/core/elastic_index.ts:134:26)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  detail:
   { index:
      { _index: '.kibana-frank-hassanabad_45',
        _id: 'alert:7652b688-a2de-404d-baee-d5b0ce4cbfa8',
        status: 400,
        error: [Object] } } }
 FATAL  Error: mapping set to strict, dynamic introduction of [interval] within [alert] is not allowed
```

which caused my instance to not start :-(

Reply:
@mikecote Any idea what might cause this?
Is this because we don't have a migration in place for alerts?

Reply:
Did Kibana migrations run @FrankHassanabad? It would be needed to update the mappings for the new field.

Reply:
They ran and then blew up there. I was already on master when I checked out this branch and ran it.

To see it yourself, update to the latest from master, and then run it. If it doesn't blow up on restart, then I would assume it might have been because I had existing alerts already.

Reply:
I can't recreate the error and looking at it it looks like what happened is that an existing alert blew up because we haven't added a migration (we're assuming there are no alerts before 7.6)